components/ProductMatrix.vue

```diff
@@ -106,11 +132,15 @@ export default Vue.extend({
 
 <style lang="sass" scoped>
 .product-matrix
-  display: flex
+  display: flex // fallback
```

Comment:
Do we need fallback?

Reply:
Not sure how it would look in a browser that doesn't support the grid. Most do (https://caniuse.com/#feat=css-grid) but it's nice to have it I guess

Reply:
Hm, some day I would like to use `grid-column-gap` and `grid-row-gap` instead of margin. I guess one need to test the support for grid in Browserstack first. Because we can't have both.